### DISCUSSION TITLE
What color scheme is being used in this gif

### DISCUSSION THREAD

**Discussion by nonetrix** [numToStr/Comment.nvim#470] [Q&A]
I really like this, even more so with how well it goes with background 
![136532939-926a8350-84b7-4e78-b045-fe21b5947388](https://github.com/numToStr/Comment.nvim/assets/45698918/aacd10a9-6558-4f6f-92c9-69bf022e5f70)

**Comment by sharpchen** [comment]
might be [rose-pine](https://github.com/rose-pine/neovim)
